thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js

```diff
@@ -597,12 +597,10 @@ export default Component.extend({
       analysisRange,
       selectedBaseline
     } = this.getProperties('metricUrn', 'analysisRange', 'selectedBaseline');
-    const granularity = '15_MINUTES';
-    const timezone = moment.tz.guess();
```

Comment:
Should we keep the timezone? @harleyjj could you check how RCA makes this call, it should use the same timezone. 

The granularity can be just left empty.

Reply:
Just checked this. 

The timezone should be pinned to "America/Los_Angeles".
Granularity should be left empty.

Thanks.

Reply:
I'll go ahead and change that asap